scripts/cnode-helper-scripts/env

```diff
@@ -1031,6 +1031,21 @@ test_koios() {
   [[ ${ENABLE_KOIOS} = 'Y' && -n ${KOIOS_API} && $(curl -Isf -m 5 ${KOIOS_API}/tip | head -1) = *"200 OK"* ]] || unset KOIOS_API
 }
 
+check_cncli_versions() {
+  if [[ $(command -v cncli) ]] ; then
+    vcur=$(cncli -V | sed 's/cncli /v/g')
+    vrem=$(curl -s https://api.github.com/repos/cardano-community/cncli/releases/latest | jq -r .tag_name)
+    if [[ ! -z ${vrem} ]]; then
+      [[ ${vcur} == ${vrem} ]] || upCNCLI="${FG_MAGENTA}CNCLI current version ($vcur) different from repo ($vrem).${NC}"
+    else
+      upCNCLI="${FG_RED}CNCLI : GITHUB API Not Reachable or Not Responding.${NC}"
+    fi
+  else
+    upCNCLI="${FG_RED}CNCLI nos installed. Run prereqs.sh -c [...] to install if needed.${NC}"
```

Comment:
We should just silently ignore for both cases, ie if github is unreachable or if cncli is not installed as this is an optional addon.

Reply:
OK sounds right indeed.